packages/expo-crypto/src/Crypto.ts

```diff
@@ -177,3 +177,45 @@ export function getRandomValues<T extends IntBasedTypedArray | UintBasedTypedArr
 export function randomUUID() {
   return ExpoCrypto.randomUUID();
 }
+
+const digestLengths = {
+  [CryptoDigestAlgorithm.SHA1]: 20,
+  [CryptoDigestAlgorithm.SHA256]: 32,
+  [CryptoDigestAlgorithm.SHA384]: 48,
+  [CryptoDigestAlgorithm.SHA512]: 64,
+  [CryptoDigestAlgorithm.MD2]: 16,
+  [CryptoDigestAlgorithm.MD4]: 16,
+  [CryptoDigestAlgorithm.MD5]: 16,
+};
+
+/**
+ * The `digest()` method of `Crypto` generates a digest of the supplied `TypedArray` of bytes `data` with the provided digest `algorithm`.
+ * A digest is a short fixed-length value derived from some variable-length input. **Cryptographic digests** should exhibit _collision-resistance_,
+ * meaning that it's very difficult to generate multiple inputs that have equal digest values.
+ * On web, this method can only be called from a secure origin (https) otherwise an error will be thrown.
```

Comment:
```suggestion
 * The `digest()` method of `Crypto` generates a digest of the supplied `TypedArray` of bytes `data` with the provided digest `algorithm`.
 * A digest is a short fixed-length value derived from some variable-length input. **Cryptographic digests** should exhibit _collision-resistance_,
 * meaning that it's very difficult to generate multiple inputs that have equal digest values.
 * On web, this method can only be called from a secure origin (HTTPS) otherwise, an error will be thrown.
```